### ISSUE TITLE
Loading animation is broken

### ISSUE THREAD

**Issue by bschoenfeld** [Code4HR/hrt-bus-finder#61]
I broke it when I added the JPanel stuff. Can you fix it, @wbprice?

**Comment by wbprice** [comment]
Now that we have a disclaimer modal, do we really need a loading animation?  I feel like we can get away with just removing it.

**Comment by qwo** [comment]
For the case of _second time someone logs on and has already seed the disclaimer_?

Are you guys deciding to put disclaimer on each load? We can skip the jQuery cookie

**Comment by wbprice** [comment]
My preference would be to have it display only once, but I'm not sure what HRT wants.  @kmcurry compared our disclaimer to a in-car navigation system, which displays a disclaimer every time.

**Comment by wbprice** [comment]
fixed in #73, the loading animation lives outside the tiny tiny box that jpanel forces everything into.

**Comment by kmcurry** [comment]
Ideally we can offer "don't show me this again" and the user can set it.

On Sat, Oct 26, 2013 at 4:35 PM, William Blaine Price <
notifications@github.com> wrote:

> My preference would be to have it display only once, but I'm not sure what
> HRT wants. @kmcurry https://github.com/kmcurry compared our disclaimer
> to a in-car navigation system, which displays a disclaimer every time.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/c4hrva/hrt-bus-finder/issues/61#issuecomment-27155175
> .

## 

Kevin Curry
Program Director
CfA Brigade
757.613.8158
GMT-4 (EDT)
http://brigade.codeforamerica.org

_Apply for Fellowship: http://codeforamerica.org/apply_

**Comment by kmcurry** [comment]
Is there a separate issue for the disclaimer? I have some comments but I can't find where to put them?

**Comment by bschoenfeld** [comment]
The original disclaimer issue is #67.
